_assets/ci/Jenkinsfile.tests

```
@@ -44,11 +44,10 @@ pipeline {
     timeout(time: getDefaultTimeout(), unit: 'MINUTES')
     disableConcurrentBuilds()
     /* manage how many builds we keep */
-    buildDiscarder(logRotator(
-      numToKeepStr: '5',
-      daysToKeepStr: '30',
-      artifactNumToKeepStr: '1',
-    ))
+    buildDiscarder(!isTestNightlyJob() ?
+        logRotator(numToKeepStr: '5', daysToKeepStr: '30', artifactNumToKeepStr: '1') :
+        logRotator(numToKeepStr: '14', daysToKeepStr: '30', artifactNumToKeepStr: '14')
+    )
```

We could make this simple:
```suggestion
    buildDiscarder(logRotator(
      numToKeepStr: '14',
      daysToKeepStr: '30',
      artifactNumToKeepStr: isTestNightlyJob() ? '14' : '1',
    ))
```

---

True :+1: , applied.